d2-milestone/src/components/question/Question.jsx

```diff
@@ -16,16 +16,16 @@ function Question() {
     const handleClose = () => setShowComponent(false);
 
     useEffect(() => {
-        axios.get('http://localhost:3001')
+        questionService.getAll()
         .then(result => {
             setQuestions(result.data);
         })
         .catch(err => console.log(err));
     }, []);
 
-    const easyQuestions = questions.filter(q => q.question_complex === "Easy");
-    const mediumQuestions = questions.filter(q => q.question_complex === "Medium");
-    const hardQuestions = questions.filter(q => q.question_complex === "Hard");
+    const easyQuestions = questions.filter(q => q.complexity == "Easy")
+    const mediumQuestions = questions.filter(q => q.complexity == "Medium")
+    const hardQuestions = questions.filter(q => q.complexity == "Hard")
```

Comment:
ESLint is flagging it to use strict typing (===). There's no issue with the code here.